tests/model_serving/model_server/kueue/test_kueue_isvc_raw.py

```
@@ -0,0 +1,123 @@
+                dyn_client=admin_client,
+            )
+        )
+        if len(deployments) != 1:
+            raise DeploymentValidationError("Too many deployments found")
+
+        deployment = deployments[0]
+        try:
+            deployment.wait_for_replicas(deployed=True, timeout=Timeout.TIMEOUT_30SEC)
+        except TimeoutExpiredError:
+            if deployment.instance.status.availableReplicas != 1:
+                raise DeploymentValidationError("Deployment should have 1 available replica")
```

Pylint (3.3.7)

[warning] Consider explicitly re-raising using 'except TimeoutExpiredError as exc' and 'raise DeploymentValidationError('Deployment should have 1 available replica') from exc'

(W0707)